tutorials/blockchains/system-chains/asset-hub/.pages

```diff
@@ -0,0 +1,5 @@
+title: Asset Hub Tutorials
```

Comment:
```suggestion
title: Asset Hub
```

Same here - I don't think tutorials is needed for the left nav as it already shows up under Blockchain Tutorials